zhaquirks/legrand/cable_outlet.py

```diff
@@ -108,5 +108,20 @@ async def write_attributes(
     QuirkBuilder(f" {LEGRAND}", " Cable outlet")
     .replaces(LegrandCluster)
     .replaces(LegrandCableOutletCluster)
+    .enum(
+        attribute_name=LegrandCluster.AttributeDefs.device_mode.name,
+        cluster_id=LegrandCluster.cluster_id,
+        enum_class=DeviceMode,
+        translation_key="device_mode",
+        fallback_name="Device mode",
+    )
+    .enum(
+        attribute_name=LegrandCableOutletCluster.AttributeDefs.pilot_wire_mode.name,
+        cluster_id=LegrandCableOutletCluster.cluster_id,
+        enum_class=PilotWireMode,
+        translation_key="pilot_wire_mode",
+        fallback_name="Pilot Wire mode",
```

Comment:
When we bump ZHA in HA, the `fallback_name` will be used to update `strings.json` in HA, so we should lowercase "wire" too.

```suggestion
        fallback_name="Pilot wire mode",
```